examples/data-objects/badge/src/helpers.ts

```diff
@@ -147,6 +149,6 @@ export const getRelativeDate = (timestamp: Date): string => {
     } else if (delta < day * 2) {
         return "yesterday";
     } else {
-        return timestamp.toUTCString();
+        return date.toUTCString();
```

Comment:
@anthony-murphy - ...until 2 days have passed, in which case we crash attempting to call ".toUTCString()" on a string.

Reply:
Hey, I wrote that bug! 😝